lib/Swagger/src/Discussion/Api/SitesApi.php

```diff
@@ -211,10 +196,6 @@ public function createSiteWithHttpInfo($site, $x_wikia_schwartz = null)
                     $data = $this->apiClient->getSerializer()->deserialize($e->getResponseBody(), '\Swagger\Client\Discussion\Models\HalProblem', $e->getResponseHeaders());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "\Swagger\Client\Discussion\Models\HalProblem" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)